doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -0,0 +1,56 @@
+
+{PRODUCT} is designed specifically to excel in a hybrid cloud environment and to be adaptable to your domain and tooling needs. The core objective of {PRODUCT} is to help you mold a set of business processes and decisions into your own domain-specific cloud-native set of services.
+
+.Business processes and decisions to cloud services
+image::kogito/kogito-assets-to-cloud.png[Image of business assets moving to cloud services]
+
+When you are using {PRODUCT}, you are building a cloud-native application as a set of independent domain-specific services, collaborating to achieve some business value. The processes and rules that you use to describe the target behavior are executed as part of the services that you create. The resulting services are highly distributed and scalable with no centralized orchestration service, and the runtime that your service uses is optimized for what your service needs.
+
+// @comment: Excluding for simplicity for now (Stetson, 18 Feb 2020)
+//If you need long-lived processes, you can persist the runtime state externally in a data grid such as Infinispan. Each {PRODUCT} service also produces events that can be consumed by other services. For example, if you use Apache Kafka, events can be aggregated and indexed in a data index service, offering advanced query capabilities through GraphQL.
+
+{PRODUCT} includes components from well-known business automation KIE projects, specifically https://drools.org[Drools], https://jbpm.org[jBPM], and https://optaplanner.org[OptaPlanner], to offer dependable, open source solutions for business rules, business processes, and constraint solving.
+
+== Cloud-first priority
+
+{PRODUCT} is designed to run and scale on a cloud infrastructure. You can use {PRODUCT} with the latest cloud-based technologies, such as Quarkus, Knative, and Apache Kafka, to get fast boot times and instant scaling on container application platforms, such as {OPENSHIFT}.
+
+.Technologies used with {PRODUCT}
+image::kogito/kogito-related-technolgies.png[Image of cloud-based technologies]
+
+For example, {PRODUCT} is compatible with the following technologies:
+
+* *{OPENSHIFT}*, based on Kubernetes, is the target platform for building and managing containerized applications.
+* *Quarkus* is the new native Java stack for Kubernetes that you can use when you build applications with {PRODUCT} services.
+* *Spring Boot* is also supported with {PRODUCT} if you need to use the Spring Framework with {PRODUCT}.
+* *GraalVM* enables you to use native compilation with {PRODUCT}, resulting in fast start-up times and minimal footprint. For example, a native {PRODUCT} service starts in about 0.003ms, about 100 times faster than a non-native start-up. Fast start-up is almost a necessity in a cloud ecosystem, especially if you need small serverless applications.
```

Comment:
maybe it's better to move up GraalVM, and take it closer to Quarkus, because that's where the real benefit for native compilation comes. We are not testing or optimizing for GraalVM with Spring Boot. Also notice that native compilation may not be supported for product in the first releases.